welcome/lib/main.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+//       theme: ThemeData.dark().copyWith(scaffoldBackgroundColor: darkBlue),
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        appBar: AppBar(
+           title: const Text('My app'),
+         ),
+        body: Center(
+          child: MyWidget(),
+        ),
+      ),
+    );
+  }
+}
+
+class MyWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+  children: <Widget>[
+    Expanded(
+              child: Text(''),
+            ),
+Expanded(
```

Comment:
Please use VSC's formatter as explained on [this slide](https://docs.google.com/presentation/d/1tybmc0YPWrdmRy72DM4TAvJq9HPVocZgZB-k9tEdNRo/edit#slide=id.g855f226bc4_9_17).

Reply:
done